optiboot/bootloaders/optiboot/optiboot.c

```diff
@@ -561,6 +577,7 @@ int main(void) {
 #if FLASHEND > 8192
 /*
  * AVR with 4-byte ISR Vectors and "jmp"
+ * WARNING: this works only up to 128KB flash!
```

Comment:
stk500v1 only works up to 128KB flash too, so this is a redundant comment WRT optiboot itself, though I guess it's fine in case someone wants to port the code elsewhere (although, are there any chips with 128K+ that don't support a boot section?)